examples/heal/nsm-system-restart/README.md

```diff
@@ -0,0 +1,145 @@
+# Nsm system restart
```

Comment:
Should we also test nsm-system restart for the local case?

Reply:
Yes, done